replica/server.go

```diff
@@ -360,6 +372,93 @@ func (s *Server) Revert(name, created string) error {
 	return nil
 }
 
+// UpdateLUNMap updates the the original LUNmap with any changes which have been
```

Comment:
hopefully this function can be called multiple times

Reply:
added comment